external/java/src/main/java/ch/epfl/dedis/lib/omniledger/contracts/DarcInstance.java

```diff
@@ -166,22 +165,6 @@ public Instruction spawnContractInstruction(String contractID, Signer s, List<Ar
         return inst;
     }
 
-    /**
-     * Like spawnContractInstruction, but creates a ClientTransaction with only this instruction and sends it
-     * to the omniledger.
-     *
-     * @param contractID the id of the contract to create
-     * @param s          the signer that is authorized to spawn this contract
-     * @param args       arguments to give to the contract
-     * @throws CothorityException
-     */
-    public TransactionId spawnContract(String contractID, Signer s, List<Argument> args) throws CothorityException {
```

Comment:
Why did you remove this? I can think of use-cases where you just want to send the spawn instruction without waiting for it to be included, no?

Reply:
I'm not sure it was on purpose. I'll add it back.